tests/Integration/Behaviour/Features/Scenario/Product/Shop/shop_management.feature

```diff
@@ -1,12 +1,12 @@
-# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags update-multi-shop-management
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags multishop_management
```

Comment:
```suggestion
# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags shop-management
```
If we follow the implied convention from other changes the tag should match the file name, so either we adapt this tag or we should rename the file itself to multishop_management.feature I guess